index.html

```diff
@@ -0,0 +1,317 @@
+  schema-->reader("👩🏽‍💻  Reader")
+  proc-->output(["📄  Output Schema"])
+```
+
+- **Authors (either human or machine)** write one or more `@using` directives into schema files to declare that the schema conforms to some set of specifications
+- **Machine processors** read the `@using` directives and compare them to a list of supported specs, activating the spec-mandated appropriate behaviors. Processors generally *pass through* schema elements they do not understand, although they may remove them or fail entirely, depending on the processor's needs and the spec's [listed purpose](#for).
+- **Human readers** see the `@using` directives and can follow the URLs to the specification, receiving an explanation of the requirements of the specification and what new directives, types, and other schema objects are available within the document.
+
+## `@using` within a document
+
+The `@using` directive is applied to `schema` declarations to declare that the schema uses a particular spec. For example,
+
+```graphql
+#::[Using the `@using` directive]
+schema @using(spec: "https://spec.example.com/example/v1.0") {
+  query: Query
+}
+```
+
+## The `@using` directive
+
+```graphql
+#::[Declaration of the `@using` directive]
+directive @using(spec: String!, prefix: String)
+  repeatable on SCHEMA
+```
+
+Documents **must** include a definition for the `@using` directive. The provided definition does not have to match the above definition exactly. Instead, it must be *compatible*. A conforming definition may:
+- **Omit optional arguments** if they are never used in the document.
+- **Introduce new directive locations**, the behavior of which is unspecified here.
+- **Introduce new arguments.** New arguments **should** be prefixed, e.g. `example_feature: Bool`. The prefix **should** be the prefix of a spec referenced with this or another `@using` directive within the document.
+
+### `spec`
+
+A URL identifying the spec. When viewed, the URL **should** provide the content of the appropriate version of the specification in some human-readable form. In short, a human reader should be able to click the link and go to the docs for the version in use. It is not required that the *content* be machine-readable in any particular way.
+
+<figure>
+  <figcaption>
+    <a name=anatomy-of-a-spec-url href=#anatomy-of-a-spec-url class=a-header>Basic anatomy of a spec URL</a>
+  </figcaption>  
+  <code class=anatomy>
+    <span class=pink style='--depth: 2'>http://spec.example.com/a/b/c/<span>exampleSpec<aside>prefix</aside></span><aside>
+      identity
+    </aside></span>/<span style='--depth: 2' class=green>v1<aside>version</aside></span>
+  </code>
+</figure>
+
+Spec URLs contain information about the spec's [prefix](#prefixing) and [version](#versioning), and should be generated and processed in accordance with the [requirements on the structure of spec URLs](#spec-urls).
+
+### `prefix`
+
+Change the prefix for schema elements from this specification.
```

Comment:
I'm not sure that you actually explicitly define outside of examples what "changing the prefix" means (ie, "replacing the default prefix from the beginning of the name of any element that starts with the default prefix and an underscore or that is exactly the default prefix" or something). I mean it's pretty obvious but probably worth spelling out.

Reply:
added language, is it clear enough?